### PR TITLE
Sheet update

### PR DESCRIPTION
**Pull request by rpassas**
Syncing changes between all the sheets

### CODE REVIEW

File: input/images/IJE_File_Layouts_Version_2021_FHIR-2023-02-22-All-Combined.csv [hunk 1]
```diff
@@ -3308,9 +3308,4 @@ Decision: Births should behave the same as deaths.",Needs work,FHIR-41387,Extens
...
 ,Fetal Death,NA,NA,NA,Record Identifier,*NO IJE MAPPING*,"YYYYJJNNNNNN,  YYYY = year JJ = jurisdiction  and NNNNNN = certificate number",,BFDR,BundleDocumentDemographicCodedContent,identifier.value,string(12),"YYYYJJNNNNNN,  YYYY = year, JJ = jurisdiction,  and NNNNNN = certificate number",,,,,,BundleDocumentDemographicCodedContent,
-,Fetal Death,NA,NA,NA,Record Identifier,*NO IJE MAPPING*,"YYYYJJNNNNNN,  YYYY = death year JJ = jurisdiction  and NNNNNN = certificate number",,BFDR,BundleDocumentCodedCauseOfFetalDeath,identifier.value,string(12),"YYYYJJNNNNNN,  YYYY = year, JJ = jurisdiction,  and NNNNNN = certificate number",,,,,,BundleDocumentCodedCauseOfFetalDeath,
-91,Mortality,NA,NA,NA,Race Recode 40  (No longer provided as of 2022),*NO IJE MAPPING*,,,VRCPL,ObservationCodedRaceAndEthnicityVitalRecords,component[ValueSetRaceRecode40VitalRecords].value,codeable,[ValueSetRaceRecode40VitalRecords],,,,,,CodedRaceAndEthnicity,
```
Comment (ptran265) [inline input/images/IJE_File_Layouts_Version_2021_FHIR-2023-02-22-All-Combined.csv:3312]: are these meant to be deleted?

File: input/pagecontent/ije_mapping_natality.md [hunk 2]
```diff
@@ -74,7 +74,6 @@ All of the none-of-the-above values are represented as observations with a clear
...
 <tr><td style='text-align: center;'>Not Mapped</td><td>Infections Present--Genital herpes simplex</td><td style='text-align: center;'>Not Mapped</td><td><a href='StructureDefinition-Condition-infection-present-during-pregnancy.html'>ConditionInfectionPresentDuringPregnancy</a></td><td>code=33839006 (Genital herpes simplex (disorder))</td><td style='text-align: center;'>na</td><td>Michigan state-specific requirement.</td></tr>
-<tr><td style='text-align: center;'>Not Mapped</td><td>Infections Present--Genital herpes simplex</td><td style='text-align: center;'>Not Mapped</td><td><a href='StructureDefinition-Condition-infection-present-during-pregnancy.html'>ConditionInfectionPresentDuringPregnancy</a></td><td>code=33839006 (Genital herpes simplex (disorder))</td><td style='text-align: center;'>na</td><td>Michigan state-specific requirement.</td></tr>
```
Comment (ptran265) [inline input/pagecontent/ije_mapping_natality.md:77]: I think Saul said this should be included since Michigan uses it

Comment (rpassas) [reply input/pagecontent/ije_mapping_natality.md:77]: it's a duplicate entry

File: input/images/IJE_File_Layouts_Version_2021_FHIR-2023-02-22-All-Combined.csv (hunk 1 above)
Comment (rpassas) [reply input/images/IJE_File_Layouts_Version_2021_FHIR-2023-02-22-All-Combined.csv:3312]: yes, they are duplicate entries

**Review by ptran265 — APPROVED** [review]

**Pull request merged**